frontend/javascripts/admin/organization/upgrade_plan_modal.tsx

```diff
@@ -87,10 +87,10 @@ function UpgradeUserQuotaModal({ destroy }: { destroy: () => void }) {
     <Modal
       title={
         <>
-          <UserAddOutlined style={{ color: "var(--ant-primary-color)" }} /> Upgrade User Quota
+          <UserAddOutlined style={{ color: "var(--ant-color-primary)" }} /> Upgrade User Quota
         </>
       }
-      okText={"Request More Users"}
+      okText={"Paid Request for more Users"}
```

Comment:
I'm not sure about the wording here. Is it clear in this modal how high the cost is? Otherwise, "Paid Request" might be a bit intimidating? :thinking:

Reply:
We get a lot of "SPAM" at the moment where people just request an upgrade not realizing that this is a paid feature.  Maybe "Request Quote for more Users"?

Reply:
> Request Quote for more Users

Yes, I like that more than "paid request..." :+1: